cookbooks/clearwater/recipes/infrastructure.rb

```diff
@@ -117,7 +117,7 @@
     cdf = "cdf." + domain
   end
 
-  ralf = if node[:clearwater][:ralf] and node[:clearwater][:ralf] > 0
+  ralf = if node[:clearwater][:ralf] and ((node[:clearwater][:ralf] == true) || (node[:clearwater][:ralf] > 0))
```

Comment:
This pull request changes the `:ralf` attribute from a count to a boolean, so we need to handle both cases.